Unknown host tools.stratio.com: unknown error

[INFO] Stratio SpaRTA ..................................... SUCCESS [  2.690 s]
[INFO] sdk ................................................ SUCCESS [ 43.946 s]
[INFO] plugins ............................................ SUCCESS [01:07 min]
[INFO] serving-core ....................................... SUCCESS [ 34.305 s]
[INFO] driver ............................................. SUCCESS [01:42 min]
[INFO] Stratio Sparta Front ............................... FAILURE [  1.095 s]
[INFO] serving-api ........................................ SKIPPED
[INFO] doc ................................................ SKIPPED
[INFO] stratio-sparta ..................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 04:12 min
[INFO] Finished at: 2016-05-02T10:40:19+08:00
[INFO] Final Memory: 55M/473M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:0.0.23:install-node-and-npm (install node and npm) on project web: Could not download Node.js: Could not download http://tools.stratio.com/buildtools/node/v0.12.2/node-v0.12.2-darwin-x64.tar.gz: Unknown host tools.stratio.com: unknown error -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]

--- a/web/pom.xml
+++ b/web/pom.xml
@@ -15,7 +15,7 @@
     <properties>
         <nodeVer>v0.12.2</nodeVer>
         <npmVer>2.7.4</npmVer>
-        <nodeURL>http://tools.stratio.com/buildtools</nodeURL>
+        <nodeURL></nodeURL>
         <customnodeDownloadRoot></customnodeDownloadRoot>
         <customnpmDownloadRoot></customnpmDownloadRoot>
         <customnpmRegistryURL></customnpmRegistryURL>
@@ -96,6 +96,7 @@
         <profile>
           <id>node</id>
           <properties>
+            <nodeURL>http://tools.stratio.com/buildtools</nodeURL>
             <customnodeDownloadRoot>${nodeURL}/node/</customnodeDownloadRoot>
             <customnpmDownloadRoot>${nodeURL}/npm/${npmVer}/</customnpmDownloadRoot>
             <customnpmRegistryURL>http://sodio.stratio.com/nexus/content/repositories/npmjs/</customnpmRegistryURL>

